play.py

```diff
@@ -15,12 +15,12 @@ def play(self):
         self.board.add_to_board(self.ships.all_ships, self.place, ship_orientation)
```

Comment:
It seems to me `board_full` and `ship_orientation` fall into configuration and might make more sense moved out of the play function.